source/jormungandr/jormungandr/instance.py

```diff
@@ -217,111 +225,133 @@ def replace_experimental_scenario(s):
 
     @property
     def journey_order(self):
+        # type: () -> Text
         instance_db = self.get_models()
         return get_value_or_default('journey_order', instance_db, self.name)
 
     @property
     def autocomplete_backend(self):
+        # type: () -> Text
         instance_db = self.get_models()
         return get_value_or_default('autocomplete_backend', instance_db, self.name)
 
     @property
     def max_walking_duration_to_pt(self):
+        # type: () -> int
```

Comment:
Just curious: Would it be possible to add it above, not in comment?
`def max_walking_duration_to_pt(self) -> int:`

Reply:
This syntax is only available on python 3, so no.